models/index.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/../config/config.json`)[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Comment:
Kindly remove the second `sequelize` initialization and throw an error if the environment is not set.

Reply:
Okay will do